### ISSUE TITLE
NTR: Skin biopsy: basement membrane thickening and duplication (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9229]
New term request Skin biopsy: basement membrane thickening and duplication (elthat_110331020821):

**Comment by pnrobinson** [comment]
@MickeySegal
Is this is referring to ECM1, and is the basement membrane is vascular, see https://pubmed.ncbi.nlm.nih.gov/16497486/ ?

**Comment by MickeySegal** [comment]
These are the same thing.  https://www.ncbi.nlm.nih.gov/books/NBK338540/ "Periodic acid-Schiff (PAS)-diastase staining shows PAS-positive, diastase-resistant basement membrane thickening and reduplication at the dermal-epidermal junction around blood vessels".  But https://www.omim.org/entry/614809 doesn't mention vascular "Renal biopsy, when performed, showed mesangial electron-dense deposits as well as thickening of the glomerular basement membrane due to intramembranous or subendothelial electron-dense deposits".

**Comment by pnrobinson** [comment]
@MickeySegal -- this finding is not terribly well documented (the paper cited in GeneReviews is not PMID and there is not much else), but I think the following is correct.


Adding new term: 
- ID: HP:6000656
- label: Papillary dermis eosinophilic hyaline material
- parents: [Abnormal dermis morphology (HP:0034924)](https://hpo.jax.org/app/browse/term/HP:0034924)
- pmid: PMID:26803878
- def: Accumulation of eosinophilic hyaline material observed around the papillary dermis and the blood vessels located in it. The papillary dermis is the uppermost layer of the dermis. It intertwines with the rete ridges of the epidermis and is composed of fine and loosely arranged collagen fibers.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9229
- synonyms:

**Comment by MickeySegal** [comment]
The term you created doesn't seem very related to what we are looking for.  

Other examples:

https://www.ncbi.nlm.nih.gov/books/NBK349072/
"Extensive reduplication of the basement membrane and associated collagen deposition within the clefts are unique ultrastructural and histopathologic findings suggestive of KS"

https://www.omim.org/entry/614809
"Renal biopsy, when performed, showed mesangial electron-dense deposits as well as thickening of the glomerular basement membrane due to intramembranous or subendothelial electron-dense deposits."

https://www.ncbi.nlm.nih.gov/books/NBK7046/
"ultrastructural examination disclosed irregular and abnormal thickening of the basement membranes of the tubules, Bowman’s capsule, and interstitial capillaries"

**Comment by pnrobinson** [comment]
Reopening.
When we make a new term, we like to have the definition be so precise that it is clear what is being referred to. Sometimes it is challenging to figure out what descriptions in papers are actually referring to, especially when the descriptions encompass many things at once.
A rule is that a user should be able to read the definition of the HPO term and decide whether or not it applies. Some of the above findings already exist in the HPO. Is there a specific finding that is missing?

**Comment by pnrobinson** [comment]
OK, closing!
